changelog/fix-action-scheduler-fatal-error

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Prevent occasional fatal erros when creating customers via ActionScheduler jobs
```

Comment:
Errors* typo

Reply:
Thanks for catching that! Fixed in 536c1fae 🙌